python/cugraph/cugraph/datasets/dataset.py

```diff
@@ -351,6 +480,33 @@ def download_all(force=False):
                     df.to_csv(save_to, index=False)
 
 
+def download_dask_all(force=False):
+    """
+    Looks in `metadata` directory and downloads all datafiles from the the URLs
+    provided in each YAML file via multiple GPUs.
+
+    Parameters
+    force : Boolean (default=False)
+        Overwrite any existing copies of datafiles.
+    """
+    default_download_dir.path.mkdir(parents=True, exist_ok=True)
+
+    meta_path = Path(__file__).parent.absolute() / "metadata"
+    for file in meta_path.iterdir():
+        meta = None
+        if file.suffix == ".yaml":
+            with open(meta_path / file, "r") as metafile:
+                meta = yaml.safe_load(metafile)
+
+            if "url" in meta:
+                filename = meta["name"] + meta["file_type"]
+                save_to = default_download_dir.path / filename
+                if not save_to.is_file() or force:
+                    blocksize = dcg.get_chunksize(meta["url"])
+                    ddf = dask_cudf.read_csv(path=meta["url"], blocksize=blocksize)
+                    ddf.to_csv(save_to, index=False)
```

Comment:
I was thinking the same suggestion here as I made for `__download_dask_csv()` - perhaps we get rid of this and just use `download_all()`, but refactor `download_all()` to simply use `urllib.request.urlretrieve()` instead of `cudf`?

Reply:
Nice suggestions! I implemented `dask_cudf` in both download functions to maintain consistency with the functions simply using `cudf`. I also realized that there might be somewhat redundant. I will commit the changes based on your suggestions.